server/src/controllers/Events/resolver.ts

```diff
@@ -373,7 +375,16 @@ ${unsubscribe}
       }
     }
 
-    return await prisma.events.update({ where: { id }, data: update });
+    return await prisma.events.update({
+      where: { id },
+      data: update,
+      include: {
+        sponsors: { include: { sponsor: true } },
+        venue: true,
+        chapter: true,
+        rsvps: { include: { user: true } },
+      },
```

Comment:
Just to make sure I get what's going on: we have to include these now because we're using them to directly update the graphql cache?

Reply:
Yes, that's how I understand it, I guess `rsvps` could be currently removed, as that's not updated after editing.